src/ActionModal/components/ActionModal.vue

```diff
@@ -903,6 +892,38 @@ export default {
         return !this.session.address
       }
     },
+    gasEstimate: {
+      query: gql`
+        query NetworkGasEstimates(
+          $networkId: String!
+          $transactionType: String
+        ) {
+          networkFees(
+            networkId: $networkId
+            transactionType: $transactionType
+          ) {
+            gasEstimate
+          }
+        }
+      `,
+      /* istanbul ignore next */
+      variables() {
+        return {
+          networkId: this.networkId,
+          transactionType: this.transactionType
+        }
+      },
+      /* istanbul ignore next */
+      update(data) {
+        if (data.networkFees) {
+          return data.networkFees.gasEstimate
+        }
+      },
+      /* istanbul ignore next */
+      skip() {
+        return !this.session.address || !this.transactionData
```

Comment:
same: use `address` getter

Reply:
I like the idea of making this reactive. Might need to check if this is not called too often (like if transactionData changes on every change of the amount input).